### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by amuamushu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This file contains Amy Nguyen's portfolio and STEP projects
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
 
 To get started:
```
Comment (billie-cat) [inline README.md:6]: I think this can be removed.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This file contains Amy Nguyen's portfolio and STEP projects
```
Comment (billie-cat) [inline README.md:3]: nit: I would add Summer 2020, just to be a bit more precise.

File: README.md (hunk 1 above)
Comment (billie-cat) [reply README.md:6]: You marked this conversation as resolved. Is there a reason why?

Comment (amuamushu) [reply README.md:6]: Sorry, I thought you were referring to a different line that was already removed. I'll go ahead and remove the lines for "To get started"

Comment (billie-cat) [reply README.md]: Not sure if you noticed, but I highlighted the lines I was referring to in the comment. This add context as to what I am doing. Another tip is to comment on the open comments before resolving them. Even if it is just to say you: Done.

**Pull request merged**